src/l10n.json

```diff
@@ -152,6 +152,8 @@
     "registration.confirmYourEmail": "Confirm Your Email",
     "registration.confirmYourEmailDescription": "If you haven't already, please click the link in the confirmation email sent to:",
     "registration.createUsername": "Create a username",
+    "registration.emailStepTitle": "What's your email?",
+    "registration.emailStepDescription": "We need this to finish creating your account. Scratch will always keep this information private.",
```

Comment:
It should also note that
1) the email must be accurate and must not be temporary
2) there is an email confirm before sharing the first project
3) the email is used for password reset

Reply:
These are good suggestions, and we'll consider them in the future when we have the chance to take another look at the language here. Thanks!